connectors/src/lib/data_sources.ts

```diff
@@ -771,7 +771,7 @@ export async function upsertTableFromCsv({
     `/data_sources/${dataSourceConfig.dataSourceId}/tables/csv`;
   const dustRequestPayload: UpsertTableFromCsvRequestType = {
     name: tableName,
-    parentId: parents[1] ?? null,
+    parentId: null, // TODO: parents[1] ?? null when we are sure parents[0] == tableId in all cases
```

Comment:
I believe notion has a two phase setup of parents. That logic of setting parentId should live in the connector IMHO as it will depend on how it is implenented?